argo_workflow_tools/dsl/dsl_decorators.py

```diff
@@ -104,6 +104,8 @@ def Task(
     env: List[k8s.EnvVar] = None,
     env_from: List[k8s.EnvFromSource] = None,
     image_pull_policy: str = None,
+    pre_task_hook: Optional[Callable[[], None]] = None,
```

Comment:
i think we can rename it to pre_hook and post_hook..
NIP

Reply:
excellent job with defining no arguments and no return code.

Reply:
thanks, I'll rename as well